template.yaml

```diff
@@ -50,6 +50,8 @@ Globals:
     Environment:
       Variables:
         NODE_OPTIONS: --enable-source-maps
+        POWERTOOLS_DEV: !If [TestEnvironment, true, false]
```

Comment:
These env vars are only added on this PR, but not the others?

Reply:
this just increases indentation when running in dev, might be worth disabling it completly as if we want to parse the messages in the future, the parsing wont work in non-test environments